src/Resources/config/services.xml

```diff
@@ -19,5 +19,14 @@
             <tag name="kernel.event_listener" event="kernel.controller" method="onKernelController" priority="-1" />
         </service>
         <service id="apy_breadcrumb_trail.annotation.listener" alias="APY\BreadcrumbTrailBundle\EventListener\BreadcrumbListener" public="true" />
+
+                <!-- Definición del servicio AnnotationReader -->
+        <service id="Doctrine\Common\Annotations\AnnotationReader" />
```

Comment:
As can be seen in the composer.json this bundle does not have a dependency on doctrine annotations, so this approach is not correct as the class potentially is not there. The service retrieval and tagging on the lines below should happen in a compiler pass instead.

Reply:
If you look the BreadcrumbLister.php file, you can see dependencia of Doctrine\Common\Annotations\Reader;

When you run the tests with the last version with symfony 6.0, we can look this message
  7x: Since symfony/framework-bundle 6.4: The "Doctrine\Common\Annotations\Reader" service alias is deprecated without replacement. It is being referenced by the "APY\BreadcrumbTrailBundle\EventListener\BreadcrumbListener" service.
    1x in BundleInitializationTest::testServicesAreRegisteredToContainer from APY\BreadcrumbTrailBundle
    1x in BreadcrumbListenerTest::testAnnotations from APY\BreadcrumbTrailBundle\EventListener
    1x in BreadcrumbListenerTest::testAttributes from APY\BreadcrumbTrailBundle\EventListener
    1x in BreadcrumbListenerTest::testMixingAnnotationsWithAttributesFails from APY\BreadcrumbTrailBundle\EventListener
    1x in BreadcrumbListenerTest::testInvokableController from APY\BreadcrumbTrailBundle\EventListener
    ...